collection.c

```diff
@@ -1134,7 +1135,11 @@ static void php_mongo_collection_insert(zval *z_collection, zval *document, zval
 		RETURN_FALSE;
 	}
 
-	if (php_mongo_api_connection_supports_feature(connection, PHP_MONGO_API_WRITE_API)) {
+	is_acknowledged = is_gle_op(z_collection, z_write_options, &link->servers->options TSRMLS_CC);
```

Comment:
I originally used `is_gle_op`, but it was a symbol conflict with the static function name :laughing: